lbrynet/wallet/manager.py

```diff
@@ -264,7 +264,12 @@ def tip_claim(self, amount, claim_id, account):
     @defer.inlineCallbacks
     def abandon_claim(self, claim_id, txid, nout):
         account = self.default_account
```

Comment:
Pass in the `account` as an argument. We need to start moving towards having account argument on all daemon commands.